src/components/Canvas/Canvas.vue

```diff
@@ -101,7 +93,7 @@ export default {
 
   methods: {
     sectionComponent (component) {
-      return require(`./Sections/${component}.vue`).default
+      return sections[component]
```

Comment:
I added the `sections` import because I couldn't get the dynamic import to work right with the other updates.